### DISCUSSION TITLE
Can't we update multiple items by passing array of items?

### DISCUSSION THREAD

**Discussion by hashan-aet** [directus/directus#13390] [Archive]
I found that the API doc has mentioned such kind of possibility at [Update Multiple Items](https://docs.directus.io/reference/items/#update-multiple-items) as same as [Create Multiple Items](https://docs.directus.io/reference/items/#create-multiple-items) but when I tried to send a PATCH request to particular collection(not a singleton one) I'm getting this error response,

```
{
    "errors": [
        {
            "message": "\"value\" must be of type object",
            "extensions": {
                "code": "FAILED_VALIDATION"
            }
        }
    ]
}

```

**Request Sample,**

`PATCH {{base_url}}/items/inventory?fields=id,stock`

```
[
    {
        "id": "38833a11-d672-11ec-962f-0242ac160003",
        "stock": 110
    },
    {
        "id": "7451c688-026a-4af2-a60c-b6a37965a82c",
        "stock": 55
    }
]
```

**Comment by rijkvanzanten** [comment]
As per the example on the right, you pass it a partial item object with the changes to apply, and either a `keys` or `query` parameter to select what items to update. It's meant to update multiple items to the same change 👍🏻

**Reply by hashan-aet** [reply]
So the API does not support updating multiple items with different changes on each item right?

**Reply by rijkvanzanten** [reply]
That's correct

**Reply by northern-mindset** [reply]
Linking a duplicated question: https://github.com/directus/directus/discussions/5530
